src/components/SearchBar.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="input-icons">
+    <input id="search" type="text" placeholder="Search" />
+    <i class="fa fa-search icon"></i>
+  </div>
+</template>
+<script>
+export default {
+  name: 'SearchBar',
+  props: {
+    Search: String,
+  },
+}
+</script>
+<style scoped>
+#search {
+  height: 30px;
+  width: 90%;
+  position: relative;
+  border: 1px solid rgba(0, 0, 0, 0.151);
+  border-radius: 30px;
+  padding-left: 15px;
+}
+#search:focus {
+  box-shadow: 1px 1px 5px grey;
+  outline: none;
+}
+#search:hover {
+  box-shadow: 1px 1px 5px grey;
+}
+
+.input-icons {
```

Comment:
You should use flexbox for this. Don't need position relative because it will confuse things later. Basically always use flexbox unless there's an incredibly good reason not to